.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
does this file need to be in the commit? I think it can be added to your global gitignore, I see it as developer specific thing. Or do you think it's a benefit to everyone?

Reply:
This was added by accident 😬